wit/messaging.wit

```diff
@@ -5,7 +5,12 @@ world imports {
 	import consumer;
 }
 
+world imports-request-reply {
+	include imports;
+	import request-reply;
+}
+
```

Comment:
Do we maybe also want a world where we `import-request-reply` and `export incoming-handler`?

```suggestion
world imports-request-reply {
	include imports;
	import request-reply;
}

world messaging-with-request-reply {
	include imports-request-reply;
	export incoming-handler;
}
```